### DISCUSSION TITLE
[✨] Add qwik add command for WebdriverIO

### DISCUSSION THREAD

**Discussion by christian-bromann** [QwikDev/qwik-evolution#157] [Proposals For Qwik]
### Is your feature request related to a problem?

Providing Playwright as the only solution for e2e testing suggests people it is fine to run tests in browser engines (some of which even modified) which by no means comes close the real browser and many folks have interest in testing in the browser that their users use.

There are whole talks about why testing in a browser engine is just not the same as testing in real browser: https://www.youtube.com/watch?v=Mo6LmFGrtxY

### Describe the solution you'd like

Similar to QwikDev/qwik#1926 I would love to help extend the qwik CLI to support the ability to use WebdriverIO.

### Describe alternatives you've considered

n/a

### Additional context

I am maintaining WebdriverIO and help at the W3C to move standards for browser automation forward. I am disappointed to see that frontend frameworks just go with the "popular" tool and disregard the development of the web platform. I am happy to step in and do the leg work. In my opinion Qwik deserves to be tested in real browser on desktop and mobile, powered by web standards and users should at least have the option to decide which tool to use.

**Comment by manucorporat** [comment]
Hi Christian!

This is awesome! i agree with you! and at qwik, we are currently in a good spot to start scaling the integrations, recently we been focusing on this!

I would recommend u to create a Qwik project (`pnpm create qwik`), then try to add WebDriverIO there, once we have that, it's easy to convert it to an integration, like this:
https://github.com/BuilderIO/qwik/tree/main/starters/features/playwright

**Comment by christian-bromann** [comment]
Sounds good. I will set up a project and raise a PR.

**Comment by PatrickJS** [comment]
this is a good first issue

**Comment by christian-bromann** [comment]
Thanks @PatrickJS , I am happy to take this one when I get the time.

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.
